util/buildkitskipper/buildkitskipper.go

```diff
@@ -0,0 +1,8 @@
+package buildkitskipper
+
+import "context"
+
+type BuildkitSkipper interface {
```

Comment:
I think we can delete this type, now that it's defined where it's used. We can probably delete this whole file.

Reply:
good catch, I forgot to remove it. done.